test/public/runs/detail.test.js

```diff
@@ -211,8 +211,64 @@ module.exports = () => {
             .to.equal('DETECTORS - CPV - A new EOR reason');
     });
 
+    it('should successfully update avgInelasticInteractionRate of PbPb run', async () => {
+        await goToPage(page, 'run-detail', { queryParameters: { runNumber: 54 } });
+        await pressElement(page, '#edit-run');
+        await fillInput(page, '#Run-avgInelasticInteractionRate input', 100.1);
+
+        await page.click('#save-run');
+        await page.waitForNetworkIdle();
+
+        await expectInnerText(page, '#Run-avgInelasticInteractionRate', 'INELavg:\n100.1');
+    });
+
+    it('should successfully update inelasticInteractionRateAtStart of PbPb run', async () => {
+        await goToPage(page, 'run-detail', { queryParameters: { runNumber: 54 } });
+        await pressElement(page, '#edit-run');
+        await fillInput(page, '#Run-inelasticInteractionRateAtStart input', 100.1);
+
+        await page.click('#save-run');
+        await page.waitForNetworkIdle();
+
+        await expectInnerText(page, '#Run-inelasticInteractionRateAtStart', 'INELstart:\n100.1');
+    });
+
+    it('should successfully update inelasticInteractionRateAtMid of PbPb run', async () => {
+        await goToPage(page, 'run-detail', { queryParameters: { runNumber: 54 } });
+        await pressElement(page, '#edit-run');
+        await fillInput(page, '#Run-inelasticInteractionRateAtMid input', 100.1);
+
+        await page.click('#save-run');
+        await page.waitForNetworkIdle();
+
+        await expectInnerText(page, '#Run-inelasticInteractionRateAtMid', 'INELmid:\n100.1');
+    });
+
+    it('should successfully update inelasticInteractionRateAtEnd of PbPb run', async () => {
+        await goToPage(page, 'run-detail', { queryParameters: { runNumber: 54 } });
+        await pressElement(page, '#edit-run');
+        await fillInput(page, '#Run-inelasticInteractionRateAtEnd input', 100.1);
+
+        await page.click('#save-run');
+        await page.waitForNetworkIdle();
+
+        await expectInnerText(page, '#Run-inelasticInteractionRateAtEnd', 'INELend:\n100.1');
+    });
+
+    it('should successfully update avgInelasticInteractionRate of pp run', async () => {
+        await goToPage(page, 'run-detail', { queryParameters: { runNumber: 49 } });
+        await pressElement(page, '#edit-run');
+        await fillInput(page, '#Run-avgInelasticInteractionRate input', 100000);
+
+        await page.click('#save-run');
+        await page.waitForNetworkIdle();
+
+        await expectInnerText(page, '#Run-avgInelasticInteractionRate', 'INELavg:\n100,000');
+        await expectInnerText(page, '#Run-muInelasticInteractionRate', '\u03BC(INEL):\n0.009');
+    });
+
```

Comment:
Idea is good, but I invite you to check how much test time you added here. Basically, goToPage takes A LOT of time, we should try to reuse the same page as much as possible. Here, I would simply put all the tests in one, or eventually 2, and if possible not reload the page.

Reply:
reduced to two tests